File: 2955-account-balance-after-rounded-purchase/2955-account-balance-after-rounded-purchase.cpp
class Solution {
public:
    int accountBalanceAfterPurchase(int purchaseAmount) {
        int rem=purchaseAmount%10;
        int rp=0;
        if(rem<5){
            rp=purchaseAmount-rem;
            return 100-rp;
        }
        rp=floor(purchaseAmount/10)*10+10;
        return 100-rp;


    
        
    }
};

File: 2955-account-balance-after-rounded-purchase/2955-account-balance-after-rounded-purchase.py
class Solution:
    def accountBalanceAfterPurchase(self, purchaseAmount: int) -> int:
        rem=purchaseAmount%10
        if(rem<5):
            r=purchaseAmount-rem
            return 100-r 
        rp=(purchaseAmount//10)*10+10
        return 100-rp
        